meson.build

```diff
@@ -25,7 +25,10 @@ pugixml_dep = dependency('pugixml', static:static_deps)
 libcurl_dep = dependency('libcurl', static:static_deps)
 microhttpd_dep = dependency('libmicrohttpd', static:static_deps)
 
-if not compiler.has_header('mustache.hpp')
+extra_include = []
+if compiler.has_header('kainjow/mustache.hpp')
+  extra_include = ['/usr/include/kainjow']
```

Comment:
`has_header` doesn't guaranty that the directory to add is `/usr/include/kainjow`. It works in debian but it is not guaranty to works all time.

It would be better, at least, to check for `mustache.hpp` but with the specific `/usr/include/kainjow` directory to be sure that it is this directory that contains the header.
This can be done with argument `args : '-I/extra/include/dir` (https://mesonbuild.com/Reference-manual.html#compiler-object)

Reply:
Done.